ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-qe-test-addon__v4.13.yaml

```diff
@@ -21,19 +21,13 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: qetest-rosa-addon
+- as: qetest-addon-aws-osd
```

Comment:
please create a separate file; do not er-use this one

Reply:
done